posts/2019-04-11-Rust-1.34.0.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: post
+title: "Announcing Rust 1.34.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.34.0. Rust is a
+programming language that is empowering everyone to build reliable and
+efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.34.0 is as easy as:
+
+```console
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the appropriate
+page on our website, and check out the [detailed release notes for 1.34.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1340-2019-04-11
+
+## What's in 1.34.0 stable
+
+The largest feature in this release is the introduction of *alternative* `cargo` registries.
+The release also includes support for `?` in documentation tests,
+some improvements for `#[attribute(..)]`s, as well as the stabilization of `TryFrom`.
+Read on for a few highlights, or see the [detailed release notes][notes] for additional information.
+
+### Alternative `cargo` registries
+
+[crates.io]: http://crates.io/
+[registry-docs]: https://doc.rust-lang.org/nightly/cargo/reference/registries.html#running-a-registry
+
+Since before 1.0, Rust has had a public crate registry, [crates.io].
+People publish crates with `cargo publish` and it's easy to include these crates
+in the `[dependencies]` section of your `Cargo.toml`.
+
+Not everyone _wants_ to publish their crates to crates.io, however.
+People maintaining proprietary/closed-source code can't use crates.io,
+and instead are forced to use `git` or `path` dependencies.
+This is usually fine for small projects, but if you have a lot of closed-source crates
+within a large organization, you lose the benefit of the versioning support that crates.io has.
+
+With this release, Cargo gains support for alternate registries.
+These registries coexist with crates.io, so you can write software that depends
```

Comment:
We might want to specify that crates on crates.io can't depend on external registries.